### PR TITLE
Some more icons (IBM, Docker, 1and1)

### PR DESCRIPTION
**Pull request by robertaramar**
A couple of more icons, which I have created security codes for and which others also might find useful.

### CODE REVIEW

File: app/src/main/res/drawable/thumb_docker.xml [hunk 1]
```diff
@@ -0,0 +1,181 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="270dp"
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_docker.xml]: Could you resize this to be max of 128x128? Alternatively docker will come in with #398

File: app/src/main/res/drawable/thumb_ibm.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="1000dp"
+    android:height="400dp"
+    android:viewportWidth="1000"
+    android:viewportHeight="400">
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_ibm.xml]: As above, the image should be at most 128x128

Comment (robertaramar) [reply app/src/main/res/drawable/thumb_ibm.xml]: done, see new push

**Comment by Naudiz** [comment]
Could you please include the finnish airline "Finnair" as a new icon aswell? They support OTP and are currently my only entry without icon in andOTP :)

**Comment by robertaramar** [comment]
> Could you please include the finnish airline "Finnair" as a new icon aswell? They support OTP and are currently my only entry without icon in andOTP :)

If you provide me with the SVG graphics, sure

**Comment by Naudiz** [comment]
The logo here is the one the company currently uses: https://commons.wikimedia.org/wiki/File:Finnair_Logo.svg

Unfortunately the current logo has no variation for 1:1 aspect ratios, but I saw andOTP also incorporates logos like Nintendo, which have the same problem.

**Comment by robertaramar** [comment]
> The logo here is the one the company currently uses: https://commons.wikimedia.org/wiki/File:Finnair_Logo.svg
> 
> Unfortunately the current logo has no variation for 1:1 aspect ratios, but I saw andOTP also incorporates logos like Nintendo, which have the same problem.

Not really a good logo for a squared icon. But anyway, I included it in the pull-request

image

**Comment by flocke** [comment]
Yeah, those kind of logos are a bit of a problem. Maybe we should just use the "F".

**Comment by robertaramar** [comment]
> Yeah, those kind of logos are a bit of a problem. Maybe we should just use the "F".

@flocke , if you want me to remove that Finnair logo from my pull-request, let me know. Please don't reject the pull-request for that. I am waiting for MY icons.

**Comment by Naudiz** [comment]
While I personally wouldn't have a problem if just the F was extracted from the logo, I don't think that this procedure would allign with Finnair's corporate identity. They seem to use the full word as logo everywhere.
I think that @robertaramar's submission is fine and should be merged. There are already far worse icons in andOTP in the regard of aspect ratio, like e.g. Nintendo, Paysafecard or Bitpanda.

File: app/src/main/res/drawable/thumb_bitmex.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="312.41dp"
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_bitmex.xml]: Can you resize this image to fit in 128x128

**Comment by robertaramar** [comment]
Hi,Yes i can, but only Friday or Saturday, I'm out of country currently and don't have access to my dev system.Regards Robert Am 29.01.2020 22:36 schrieb Richy HBM <notifications@github.com>:@RichyHBM requested changes on this pull request.



In app/src/main/res/drawable/thumb_bitmex.xml:
> @@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="312.41dp"

Can you resize this image to fit in 128x128

**Pull request merged**